src/measurement.rs

```diff
@@ -35,16 +38,9 @@
 /// // You should't need it in your own code.
 /// fn main() { }
 /// ```
-
-#[cfg(feature = "no-std")]
-use core as std;
-
-#[cfg(feature = "no-std")]
-use core::num::Float;
-
-/// All measurements implement this.
 ///
-/// It provides conversion functions to and from raw numbers.
+/// *Note*: If you are in a `no_std` environment, you have to
```

Comment:
We could probably replace all the std:: uses with core:: - that is available for libstd users.